baccmod/grid3d_acceptance_map_creator.py

```diff
@@ -191,7 +194,7 @@ def f(*args):
             return -np.sum(
                 log_poisson(count_map, fnc(x, y, *args) * exp_map / exp_map_total, log_factorial_count_map))
 
-        logger.info(f"seeds :\n{seeds}")
+        logger.log(MOREINFO, f"seeds :\n{seeds}")
```

Comment:
I would put it as debug, it's very low level information.

Reply:
Done